src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/RMSMappingQuality.java

```diff
@@ -124,6 +130,13 @@ private String makeRawAnnotationString(final List<Allele> vcAlleles, final Map<A
             rawMQdata = vc.getAttributeAsString(getRawKeyName(), null);
         }
         else if (vc.hasAttribute(getDeprecatedRawKeyName())) {
+            if (!allowOlderRawKeyValues) {
+                throw new UserException.BadInput("Presence of '-"+getDeprecatedRawKeyName()+"' annotation is detected. This GATK version expects key "
+                        + getRawKeyName() + " with an long tuple of sum of squared MQ values and total reads over variant "
+                        + "genotypes as the value. This could indicate that the provided input was produced with an older version of GATK. " +
+                        "Use the argument '--"+RMS_MAPPING_QUALITY_OLD_BEHAVIOR_OVERRIDE_ARGUMENT+"' to override and attempt the deprecated MQ calculation.");
```

Comment:
Should you add a warning that this is unlikely to work, and is likely to produce incorrect results?

Reply:
It *should* be able to produce correct results, but I must have screwed something up. Clearly it doesn't work 100% of the time, so that warning is a great idea. I'll put it on my tech debt list to see if I can fix it.